src/Microsoft.AspNet.Mvc.TagHelpers/LinkTagHelper.cs

```diff
@@ -214,7 +235,7 @@ private void BuildGlobbedLinkTags(IDictionary<string, string> attributes, TagHel
 
             foreach (var url in urls)
             {
-                attributes["href"] = HtmlEncoder.HtmlEncode(url);
+                attributes["href"] = url;
```

Comment:
why did this change?  if URLs are getting double-encoded, submit a quick fix and add a test for that -- separately